### PR TITLE
Hopefully start making the generated source a little more approachable

### PR DESCRIPTION
**Pull request by thomasvl**

### CODE REVIEW

**Comment by thomasvl** [comment]
Note: in the _Files changed_ view, using the _### files_ dropdown to filter to "Sources/protoc-gen-swift" might help see the changes that were needed to explain the diffs in the generated files themselves.  Some ivars had to become `fileprivate` so the extension has access to them.

Assuming this makes sense, there's other things we can continue to move down, this being those two protocols happened to make for a logical checkpoint.

File: Sources/protoc-gen-swift/MessageGenerator.swift [hunk 1]
```diff
@@ -167,7 +164,7 @@ class MessageGenerator {
     }
   }
 
-  func generateNested(printer p: inout CodePrinter, file: FileGenerator, parent: MessageGenerator?) {
+  func generateMainStuct(printer p: inout CodePrinter, file: FileGenerator, parent: MessageGenerator?) {
```
Comment (tbkka) [inline Sources/protoc-gen-swift/MessageGenerator.swift]: Should be `generateMainStruct` (with an 'r')

File: Sources/protoc-gen-swift/MessageGenerator.swift [hunk 2]
```diff
@@ -314,6 +295,44 @@ class MessageGenerator {
     }
   }
 
+  func generateRuntimeSupport(printer p: inout CodePrinter, file: FileGenerator, parent: MessageGenerator?) {
```
Comment (tbkka) [inline Sources/protoc-gen-swift/MessageGenerator.swift]: I presume the next PR would split this further into `generateProtoNameProviding` and `generateMessageImplementationBase` or some such?

File: Sources/protoc-gen-swift/MessageGenerator.swift [hunk 3]
```diff
@@ -314,6 +295,44 @@ class MessageGenerator {
...
+    p.print("\n")
+    p.print("// Support for the runtime.\n")
```
Comment (tbkka) [inline Sources/protoc-gen-swift/MessageGenerator.swift]: This comment seems gratuitous.

File: Sources/protoc-gen-swift/MessageGenerator.swift (hunk 1 above)
Comment (thomasvl) [inline Sources/protoc-gen-swift/MessageGenerator.swift]: Doh, I stared that that a few times saying something was wrong, but failed to notice it.  :(

File: Sources/protoc-gen-swift/MessageGenerator.swift (hunk 3 above)
Comment (thomasvl) [inline Sources/protoc-gen-swift/MessageGenerator.swift]: Removed.  I was also debating just doing one comment before all these later decls (from the FileGenerator), but I'm not sure it's worth it, so just removing it.

File: Sources/protoc-gen-swift/MessageGenerator.swift (hunk 2 above)
Comment (thomasvl) [inline Sources/protoc-gen-swift/MessageGenerator.swift]: Hadn't really thought about it, but makes sense.

**Pull request merged**